hack/local-setup.sh

```diff
@@ -171,12 +188,17 @@ deployExternalDNS ${KIND_CLUSTER_CONTROL_PLANE}
 #6. Deploy argo cd
 deployArgoCD ${KIND_CLUSTER_CONTROL_PLANE}
 
-#7. Deploy Dashboard
+#7. Deploy webhook conifgs
+deployWebhookConfigs ${KIND_CLUSTER_CONTROL_PLANE}
```

Comment:
I might be wrong but my understanding is that we will only need the webhook configs in the workload clusters

Reply:
Yeah, this makes sense to me. Webhook in the workload clusters, endpoint in the control plane. (Though we might want webhooks for gateways on the control plane in the future)

Reply:
ok cool that makes sense, i misunderstood the flow of the script ill change that